commit-message.md

```diff
@@ -0,0 +1,36 @@
+# Guidelines to create great commit messages
+
+
+We use [GitCop](https://gitcop.com) to check that commit messages are
+properly written. The rules are the following:
+
+* The first line of a commit message, called the subject line should
+  not be more than 80 characters long.
```

Comment:
i thought it was supposed to be:
- first line max 50 chars
- all other lines, soft max of 72 chars (links break this, for example)

what does the git project use?